addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -102,6 +103,7 @@ const WebsiteWysiwyg = Wysiwyg.extend({
      * @override
      */
     destroy: function () {
+        socialMediaOptions.clearDbSocialValuesCache();
```

Comment:
Wouldn't it be worth a small comment to hint why we need this, and why we need this here?
Mentioning that we need a cache during the page edition to not fire multiple RPCs each time we play with the social media snippet, but we don't need a cache living longer that this (and especially not through page navigation / during the whole tab life) to avoid outdated cache while someone else / you on another tab could have edited those website fields.

As a reminder, this is more critical than keeping an outdated cache of snippets and/or custom snippets as in that case, it will "just" be some missing data (a new custom snippet) and not corrupted/incorrect data (old/wrong social media value)